test/integration/smoke/test_public_ip_range.py

```diff
@@ -377,8 +393,45 @@ def test_dedicate_public_ip_range_for_system_vms_02_cpvm(self):
             "vlan":self.services["vlan"]
         }
 
-        self.base_system_vm(
-            services,
-            'consoleproxy'
-        )
+        try:
+            self.base_system_vm(
+                services,
+                'secondarystoragevm'
```

Comment:
copy and paste fail here.  This should be the consoleproxy being tested in this test
```suggestion
                'consoleproxy'
```